content/en/platform/corda/5.0-alpha/deploying/overview.md

```diff
@@ -7,4 +7,7 @@ menu:
     weight: 3050
 section_menu: corda-5-alpha
 ---
-This section describes the options for Corda 5 cluster deployments and contains tutorials to guide you through the process.
+Corda 5 Alpha is intended for deployment to a Kubernetes cluster using Amazon Elastic Kubernetes Service (EKS).
+As part of the release, R3 delivers Docker images and Helm charts to aid the deployment process.
+For each deployment, you must create a YAML file to define a set of Helm overrides for that environment.
+The deployment and its configuration process is described [here](../deployment-tutorials/tutorial-k8s.html) in the Tutorials section.
```

Comment:
```suggestion
The deployment and its configuration process is described in the [Deploying to a Local Kubernetes Cluster](../deployment-tutorials/tutorial-k8s.html) tutorial.
```
It's best practice to avoid generic words like "here", "more", "click here" in hyperlinks. Use the actual title of the doc you're linking to.